src/test/scala/com/mongodb/spark/sql/DataFrameReaderSpec.scala

```diff
@@ -16,132 +16,7 @@
 
 package com.mongodb.spark.sql
 
-import com.mongodb.spark.config.ReadConfig
```

Comment:
The core of the tests have just been moved into a new trait so they can be shared.